runtime/src/test/identity.rs

```diff
@@ -69,6 +71,76 @@ fn only_claim_issuers_can_add_claims() {
     });
 }
 
+#[test]
+fn only_claim_issuers_can_add_claims_batch() {
+    with_externalities(&mut build_ext(), || {
+        let owner_did = register_keyring_account(AccountKeyring::Alice).unwrap();
+        let issuer_did = register_keyring_account(AccountKeyring::Bob).unwrap();
+        let issuer = AccountKeyring::Bob.public();
+        let claim_issuer_did = register_keyring_account(AccountKeyring::Charlie).unwrap();
+        let claim_issuer = AccountKeyring::Charlie.public();
+        let mut claim_records = Vec::new();
+        let claim_key = "key".as_bytes();
+        claim_records.push(ClaimRecord::new(
+            claim_issuer_did.clone(),
+            claim_key.to_vec(),
+            100u64,
+            ClaimValue {
+                data_type: DataTypes::VecU8,
+                value: "value 1".as_bytes().to_vec(),
+            },
+        ));
+        claim_records.push(ClaimRecord::new(
+            claim_issuer_did.clone(),
+            claim_key.to_vec(),
+            200u64,
+            ClaimValue {
+                data_type: DataTypes::VecU8,
+                value: "value 2".as_bytes().to_vec(),
+            },
+        ));
+        assert_ok!(Identity::add_claims_batch(
+            Origin::signed(claim_issuer.clone()),
+            claim_issuer_did.clone(),
+            claim_records,
+        ));
```

Comment:
This confirms that the function was executed successfully but does not check if the function did what it's supposed to do (i.e. add claims).

I know most of our current test cases do the same but we're slowly working towards a better testing system. Please also fetch the claims from storage after executing this function and use `assert_eq!` to make sure they are the same as what you passed to this function.

`let claim = Identity::claims(($did, $claim_meta_data));` should fetch the claim from storage.

Reply:
I added storage checks in the unit test function.